slips_files/common/parsers/arg_parser.py

```diff
@@ -108,7 +110,8 @@ def get_configfile(self):
 
     def parse_arguments(self):
         # Parse the parameters
-        slips_conf_path = os.path.join(os.getcwd(), "config/slips.conf")
+        # slips_conf_path = os.path.join(os.getcwd(), "config/slips.conf")
```

Comment:
hey feel free to delete code that is no longer needed instead of commenting it :)) we can always find it again if necessary